AdminWebsite/AdminWebsite/ClientApp/src/app/suitability/services/questionnaire-api.service.ts

```diff
@@ -0,0 +1,48 @@
+import { ParticipantQuestionnaire, SuitabilityAnswerGroup, SuitabilityAnswer } from './../participant-questionnaire';
+import { ParticipantSuitabilityAnswerResponse, SuitabilityAnswerResponse } from './../../services/clients/api-client';
+import { BHClient } from 'src/app/services/clients/api-client';
+import { Injectable } from '@angular/core';
+import { ScrollableSuitabilityAnswersService, SuitabilityAnswersPage } from './scrollable-suitability-answers.service';
+
+@Injectable()
+export class QuestionnaireApiService implements ScrollableSuitabilityAnswersService {
+    constructor(private client: BHClient) {}
+
+    async getSuitabilityAnswers(cursor: string, limit: number): Promise<SuitabilityAnswersPage> {
+        const response = await this.client.getSuitabilityAnswers(cursor, limit).toPromise();
+        const page = new SuitabilityAnswersPage();
+        page.nextCursor = response.next_cursor;
+        page.questionnaires = response.participant_suitability_answer_response.map(item => this.map(item));
+        return page;
+    }
+
+    private map(response: ParticipantSuitabilityAnswerResponse): ParticipantQuestionnaire {
+        return new ParticipantQuestionnaire({
+            displayName: `${response.first_name} ${response.last_name}`,
+            caseNumber: response.case_number,
+            hearingRole: response.hearing_role,
+            representee: response.representee,
+            hearingId: 'missing',
```

Comment:
Should we delete this if it's not part of the response from the BFF?

Reply:
Yes, we don't use it